shared-data/default-theme/html/jsapi/global/global.js

```diff
@@ -5,38 +5,12 @@ Mailpile.fix_url = function(url) {
   return url;
 }
 
-Mailpile.go = function(url) {
```

Comment:
You are missing the idea that autoajax is, technically, optional. Please back these changes out so things continue to work without autoajax. :-)